src/procedures/CancelSTO.ts

```diff
@@ -0,0 +1,73 @@
+import { Procedure } from './Procedure';
+import { ProcedureTypes, PolyTransactionTags, CancelSTOArgs, ErrorCodes } from '../types';
+import { PolymathError } from '../PolymathError';
+import { isValidAddress } from '../utils';
+
+export class CancelSTO extends Procedure<CancelSTOArgs> {
```

Comment:
The procedure should be renamed to `CancelSto` (casing)

Remember to rename the file as well